internal/env/env.go

```diff
@@ -78,7 +78,7 @@ func PrepareRunnerEnv(cfg *config.Config) []string {
 	allowedEnvs := append(defaultEnvs, cfg.Runner.AllowedEnv...)
 
 	runnerEnv := allowedOnly(allowedEnvs)
-	runnerEnv = append(runnerEnv, "N8N_RUNNERS_SERVER_ENABLED=true")
+	runnerEnv = append(runnerEnv, "N8N_RUNNERS_RUNNER_HEALTH_CHECK_SERVER_ENABLED=true")
```

Comment:
`N8N_RUNNERS_RUNNER_` looks almost like a mistake. Why not `N8N_RUNNERS_HEALTH_CHECK_SERVER_ENABLED`? We don't add this `_RUNNER_` infix to other env vars passed to the runner like `N8N_RUNNERS_AUTO_SHUTDOWN_TIMEOUT`.